SPECS/efivar/efivar.spec

```diff
@@ -1,13 +1,15 @@
 Summary:        Tools and libraries to manipulate EFI variables
 Name:           efivar
-Version:        37
-Release:        6%{?dist}
+Version:        39
+Release:        1%{?dist}
 License:        LGPL-2.1
 Vendor:         Microsoft Corporation
 Distribution:   Azure Linux
 Group:          System Environment/System Utilities
 URL:            https://github.com/rhboot/efivar
-Source0:        https://github.com/rhboot/efivar/releases/download/%{version}/%{name}-%{version}.tar.bz2
+Source0:        https://github.com/rhboot/efivar/archive/refs/tags/%{version}.tar.gz#/%{name}-%{version}.tar.gz
```

Comment:
I noticed that the updated Source0 is different from the new downloadUrl; is that correct?

Reply:
No. I think the `#` was meant to start a comment, but it actually ended up making an irrelevant change to the URL...

Reply:
Actually, I think it was right. The `#` caused the downloaded file to have the name `efivar-39.tar.gz` which is needed for the blob storage